### DISCUSSION TITLE
How do I make endpoint publicly available without jwt token?

### DISCUSSION THREAD

**Discussion by kingRayhan** [pocketbase/pocketbase#23] [Q&A] [answered]
I saw every collection/endpoint needs an admin or user token. How do I make the endpoint publicly available without jwt token?

**Comment by ganigeorgiev** [comment, marked as answer]
Open the Collection settings panel -> API Rules -> and unlock the action rules you want to specify a custom rule/filter.

Leaving it empty is equivalent to no filter/access check.

You could also find more info in the Manage collections guide - https://pocketbase.io/docs/manage-collections/#collection-api-rules
